x-element.js

```diff
@@ -344,15 +344,8 @@ export default class XElement extends HTMLElement {
         throw new Error(`Unexpected value for "${path}.${subKey}" (expected Boolean, got ${typeName}).`);
       }
     }
-    if (!internal && RESERVED_PROPERTY_NAMES.has(key)) {
-      const { xElementPropertyName, reservedAttribute } = RESERVED_PROPERTY_NAMES.get(key);
-      if (xElementPropertyName) {
-        throw new Error(`Unexpected key "${path}" shadows in XElement.prototype interface.`);
-      } else if (reservedAttribute) {
-        console.warn(`Unexpected key "${path}" shadows related reserved attribute "${reservedAttribute}", behavior not guaranteed.`); // eslint-disable-line no-console
-      } else {
-        console.warn(`Unexpected key "${path}" shadows reserved interface, behavior not guaranteed.`); // eslint-disable-line no-console
-      }
+    if (!internal && XElement.__prototypeInterface.has(key)) {
```

Comment:
If I understand correctly, this guards against accidentally stomping XElement internal names?